docs/rules/no-document-during-ssr.md

```diff
@@ -0,0 +1,34 @@
+# Disallow references to document during SSR (`lwc/no-document-during-ssr`)
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
```

Comment:
How was this generated? Is the generated header consistent with the rest of the docs?

Reply:
In the original repo, these were generated files & updated with a commit hook. Looks like that's not the case here, so we'll want to 1) hand-craft these, 2) add back the automation, or 3) remove them if we don't want to invest in docs & to avoid stale documentation.

Reply:
I'll just make them consistent with the rest of the repo and add automation (if required) as in a separate PR.